src/utils/localization/fetchLocaleData.ts

```diff
@@ -0,0 +1,11 @@
+import { getAssetPath } from "@stencil/core";
+
+export async function fetchLocaleStrings(locale: string, component: string): Promise<any> {
+  const res = fetch(getAssetPath(`./assets/${component}/t9n/${locale}.json`))
+    .then((resp) => resp.json())
+    .catch(() => {
+      console.error(`Translations for "${locale}" not found or invalid, falling back to english`);
```

Comment:
I don't think we need to warn here since we'll always normalize to a supported locale.